proto/gossip/v1/gossip.proto

```diff
@@ -182,6 +183,9 @@ message ChainGovernorStatus {
     string emitter_address = 1; // human-readable hex-encoded (leading 0x)
     uint64 total_enqueued_vaas = 2;
     repeated EnqueuedVAA enqueued_vaas = 3; // Only the first 20 will be included.
+    int64 small_tx_net_notional_value = 4;
```

Comment:
Just confirming, should this be here instead of under `Chain` alongside `remaining_available_notional`? Does each emitter have it's own set of notional values in a flow cancel world?

Reply:
Yeah maybe under `Chain` makes more sense especially since that's where the existing `notional` measure is. I'll add a new commit.

Reply:
Okay, changing my review to request changes in the interim